lbz/jwt_utils.py

```diff
@@ -9,25 +9,20 @@
 from jose import jwt
 from jose.exceptions import JWTError, JWTClaimsError, ExpiredSignatureError
 
-from lbz.exceptions import Unauthorized
+from lbz.exceptions import Unauthorized, SecurityError
 from lbz.misc import get_logger
 
 logger = get_logger(__name__)
 
 PUBLIC_KEYS: List[dict] = []
 ALLOWED_AUDIENCES = []
+ALLOWED_ISS = os.environ.get("ALLOWED_ISS", "")
 
 if allowed_pubkeys_str := os.environ.get("ALLOWED_PUBLIC_KEYS"):
     PUBLIC_KEYS.extend(json.loads(allowed_pubkeys_str)["keys"])
 if allowed_audiences_str := os.environ.get("ALLOWED_AUDIENCES"):
     ALLOWED_AUDIENCES.extend(allowed_audiences_str.split(","))
 
-if any("kid" not in public_key for public_key in PUBLIC_KEYS):
```

Comment:
I would keep this condition (only this one) here. It confirms only if all public keys are correct. So, it can be done once at the very beginning because if `PUBLIC_KEYS` is empty, nothing bad will happen 😉